src/test/rpc/Transaction_test.cpp

```diff
@@ -774,6 +776,74 @@ class Transaction_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testBinaryRequest(unsigned apiVersion)
+    {
+        testcase(
+            "Test binary request API version " + std::to_string(apiVersion));
+
+        using namespace test::jtx;
+        using std::to_string;
+
+        Env env{*this};
+        Account const alice{"alice"};
+        Account const gw{"gw"};
+        auto const USD{gw["USD"]};
+
+        env.fund(XRP(1000000), alice, gw);
+        std::shared_ptr<STTx const> txn = env.tx();
```

Comment:
Could be `std::shared_ptr<STTX const> const txn`

Reply:
Thanks, done